code/modules/integrated_electronics/subtypes/output.dm

```diff
@@ -225,7 +223,7 @@
 
 /obj/item/integrated_circuit/output/video_camera
 	name = "video camera circuit"
-	desc = "This small camera allows a remote viewer to see what it sees."
+	desc = "Takes a string as a name and a boolean to determine whether it is on, and uses this to be a Camera, linked to the Research network."
```

Comment:
Lowercase camera. Also this seems awkwardly worded.

Reply:
it's a weird one to explain, yeah, but it's better than the previous one

Reply:
lowercase camera and research